<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeTableStatisticss</class>
 <widget class="QDialog" name="ChangeTableStatisticss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="labelData">
         <property name="text">
          <string>Плочаток роботи</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="BoxCosts"/>
       </item>
       <item>
        <widget class="QLabel" name="labelCosts">
         <property name="text">
          <string>Витрати</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="BoxProfit"/>
       </item>
       <item>
        <widget class="QLabel" name="labelProfit">
         <property name="text">
          <string>прибуток</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ButtonApply">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:#00AC14;
color:#FFE53E;
border:0px solid #FFFFFF;

}
QPushButton:hover{
background:#2DBE3B;
color:#FFE53E;
border:0px solid #FFFFFF;
}
QPushButton:pressed{
background:qlineargradient(spread:reflect, x1:0.433, y1:0, x2:0.438494, y2:0.500227, stop:0 rgba(255, 255, 255, 255), stop:0.365169 rgba(49, 196, 64, 255));
color:#FFE53E;
border:0px solid #FFFFFF;
}</string>
       </property>
       <property name="text">
        <string>Підтвердити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonClose">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:#00AC14;
color:#FFE53E;
border:0px solid #FFFFFF;

}
QPushButton:hover{
background:#2DBE3B;
color:#FFE53E;
border:0px solid #FFFFFF;
}
QPushButton:pressed{
background:qlineargradient(spread:reflect, x1:0.433, y1:0, x2:0.438494, y2:0.500227, stop:0 rgba(255, 255, 255, 255), stop:0.365169 rgba(49, 196, 64, 255));
color:#FFE53E;
border:0px solid #FFFFFF;
}</string>
       </property>
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLabel" name="labelReport">
       <property name="text">
        <string>Звіт</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QTextEdit" name="textEditReport"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
